src/main/java/com/kobylynskyi/graphql/codegen/model/request/GraphQLRequestSerializer.java

```diff
@@ -0,0 +1,60 @@
+package com.kobylynskyi.graphql.codegen.model.request;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class GraphQLRequestSerializer {
+
+    public static String serialize(GraphQLRequest graphQLRequest) {
+        if (graphQLRequest == null || graphQLRequest.getRequest() == null) {
+            return null;
+        }
+        StringBuilder builder = new StringBuilder();
+        builder.append("{\"query\":\"");
```

Comment:
I believe the JSON construction here should be separate from building the query string (and maybe use a library although this is debatable).
GraphQL requests actually expect the `operationName`, `query`, and `variables` JSON fields AFAIR, maybe we should include that as well.

Reply:
I know for sure that `operationName`, `query`, `variables` is not mandatory.
The following query works on one of my projects:
```json
{"query":"query { eventsByIds(ids: [ "4", "5", "6" ]) { id } }"}
```
But maybe some other graphql server will expect these parameters. So I will include them. I will take a note and make a change in the upcoming PR. Thanks!